packages/sdk-js/src/GrowthBook.ts

```diff
@@ -610,7 +611,20 @@ export class GrowthBook<
   public evalFeature<
```

Comment:
todo: make this _private, with a public thin wrapper. Public should not expose the ctx